modules/cij/runner.py

```diff
@@ -430,8 +430,8 @@ def trun_to_junitfile(trun, fpath=None):
                     doc_failure = doc.createElement("failure")
                     doc_failure.setAttribute(
                         "message",
-                        "not executed" if tcase.rcode is None
-                        else "test failed"
+                        "not executed" if tcase.rcode is None else
+                        "test failed"
```

Comment:
Ah sorry, I wasn't clear enough on this one; I meant

```python
"not executed" if tcase.rcode is None
                        else "test failed"
```
but the version you just posted actually works quite well!